### ISSUE TITLE
Upgrade Heroku Stack

### ISSUE THREAD

**Issue by rubillionaire** [risd/risd-webhook-form-processing#1]
Heroku is deprecating the `heroku-16` stack that this application currently uses on May 1st, 2021. In order to continue making changes to this repository, the stack must be upgraded following the [upgrade guide][heroku-stack-upgrade-guide]. Follow the directions there and once the app stack is upgraded, this issue can be considered resolved.

[heroku-stack-upgrade-guide]:https://devcenter.heroku.com/articles/upgrading-to-the-latest-stack

**Comment by rubillionaire** [comment]
this has been updated. part of the move to `heroku-20` from `heroku-16` was a new set of python versions that were available for use. this meant i had to update our python version to `python-3.6.13`. i was able to do that locally on my computer in order to test & verify that our python process which coordinates everything. to pin this python version `runtime.txt` was added, which tells heroku which python version to use. for local use `.python-version` was added to the repository which can be used with `pyenv` to define a local development environment.

`heroku-20` the version we migrated to is supported through April 2025.
